modules/admin_manual/pages/installation/manual_installation/script_guided_install.adoc

```diff
@@ -80,23 +59,25 @@ This is the name of the root user, usually `root`.
 
 == Usage
 
-It is mandatory to run this script with root previleges, because files and directories
-will be created and ownwership and permissions will be set. Call the instance script with the name you chose:
+It is mandatory to run this script with root previleges, because files and directories will be created and ownwership and permissions will be set. Call the instance script like:
 
 [source,console]
 ----
 sudo ./instance.sh
 ----
 
-The script asks you a couple of questions which follow a logical path. Default
-answers are capitalized. When the script extracts the files with tar, it automatically
-extracts them to the target location without copying. In case of an upgrade, the old
-instance path is backed up by renaming and adding a time stamp. A new target folder with
-the old name is created. This ensures that in case of issues, you can easily go back to the previous version.
-After a successful upgrade, you must manually remove the backup folder.
+The script asks you a couple of questions which follow a logical path. Default answers are capitalized.
+
+.When installing
+the script extracts the files with tar and automatically extracts them to the target location without copying.
+
+.In case of an upgrade
+the old instance path is backed up by renaming and adding a time stamp. A new target folder with the old name is created. This ensures in case of extracting and preperation issues, you can easily go back to the previous version. After a successful upgrade, you must manually remove the backup folder.
```

Comment:
```suggestion
the old instance path is backed up by renaming and adding a time stamp. A new target folder with the old name is created. This ensures that in case of extracting and preparation issues, you can easily go back to the previous version. After a successful upgrade, you must manually remove the backup folder.
```